evergreen/make_docs.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+DIR="$PWD"
+
+if [[ $(lsb_release -is 2>/dev/null) == "Ubuntu" ]]; then
+    sudo apt install texlive texlive-latex-extra texlive-fonts-recommended -y
+else
+    echo "skipping installation of deps for non-Ubuntu OS"
+fi
+
+MKFILE="$DIR/docs/overview.md"
```

Comment:
Why "MKFILE" for this name? Sounds like that implies some kind of `make` file

Reply:
Oh no reason, just a name!